docs/content/docs/recipes/teams.md

```diff
@@ -0,0 +1,347 @@
+2. How to run and connect your Teams application to a Fluid service (Azure Fluid Relay)
+3. How to create and get Fluid Containers and pass it to a React component
+
+For an example of how this recipe may be used to build a more complex application, check out the [Teams Fluid Hello World](https://github.com/microsoft/FluidExamples/tree/main/teams-fluid-hello-world) in our FluidExamples repo.
+
+{{< callout note >}}
+
+This tutorial assumes that you are familiar with the [Fluid Framework Overview]({{< relref "/docs/_index.md" >}}) and that you have completed the [QuickStart]({{< relref "quick-start.md" >}}). You should also be familiar with the basics of [React](https://reactjs.org/), [React Hooks](https://reactjs.org/docs/hooks-intro.html), and [Microsoft Teams Tabs](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/what-are-tabs).
+
+{{< /callout >}}
+
+## Create the project
+
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project, e.g., `/My Microsoft Teams Projects`.
+1. Create a vanilla Teams tab application by running the below command. See [the Microsoft Teams documentation](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/create-channel-group-tab?tabs=nodejs) for more information on setting up Teams applications.
+
+    ```dotnetcli
+    yo teams
+    ```
+
+1. After creating the project, navigate to it with the command `cd <your project name>`.
+1. The project will use three Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    | `fluid-framework`    |Contains the IFluidContainer and other [distributed data structure]({{< relref "dds.md" >}}) that synchronizes data across clients.|
+    | `@fluidframework/azure-client`   |Defines the starting schema for the [Fluid container][].|
+    | `@fluidframework/test-client-utils` |Defines the `InsecureTokenProvider` needed to create the connection to a Fluid service.|
+    {.table}
+
+    Run the following command to install the libraries.
+
+    ```dotnetcli
+    npm install @fluidframework/azure-client fluid-framework @fluidframework/test-client-utils
+    ```
+
+## Code the project
+
+1. Open the file `/src/client/<your tab name>` in your code editor. Create a new file named `Util.ts`. Then add the following import statements:
+
+    ```ts
+    //`Util.ts
+
+    import { IFluidContainer } from "fluid-framework";
+    import { AzureClient, AzureClientProps, LOCAL_MODE_TENANT_ID } from "@fluidframework/azure-client";
+    import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+    ```
+
+### Defining Fluid functions and parameters
+
+1. Since this app is intended to be used in the context of Microsoft Teams, having all Fluid related imports, initialization, and functions in one place makes it easier to use in the future. Add the following code below the import statements. The comments will define all the functions and constants needed to interact with the Fluid service and container.
```

Comment:
```suggestion
1. Since this app is intended to be used in the context of Microsoft Teams, having all Fluid-related imports, initialization, and functions in one place makes it easier to use in the future. Add the following code below the import statements. The comments will define all the functions and constants needed to interact with the Fluid service and container.
```